ui/src/pages/backoffice/Acquisition/Order/OrderEditor/components/OrderForm/OrderForm.js

```diff
@@ -16,22 +16,26 @@ import _has from 'lodash/has';
 const orderSubmitSerializer = (values, newRecord) => {
   const submitValues = { ...values };
   if (newRecord) {
-    submitValues.created_by_pid = sessionManager.user.id;
+    submitValues.created_by_pid = String(sessionManager.user.id);
```

Comment:
did you check if it is possible to cast the `user.id` to string when it's fetched so we don't have to do it in every place, like here? because we need it on other places too

Reply:
@kprzerwa I will revert this, in favour of the upcoming change where we cast them all in the `api`.

Reply:
This is what we discussed yesterday. Now re-thinking about that, while I think it would be more correct to keep it as an `int` and cast to String when using REST endpoints that require this field as String, it is probably just easier to always get and send String, otherwise every time we read a loan/purchase/ill, etc. in JS, we will have to cast it to `int`.